packages/components/src/custom-select-control/types.ts

```diff
@@ -24,7 +24,7 @@ export type CustomSelectOption = {
 /**
  * The object returned from the onChange event.
  */
-type CustomSelectChangeObject< T extends CustomSelectOption > = {
+export type CustomSelectChangeObject< T extends CustomSelectOption > = {
```

Comment:
Better if we could remove this export, as noted above.

Reply:
Removed